### ISSUE TITLE
Add Fluid Tank

### ISSUE THREAD

**Issue by CrafterKina** [CrafterKina/Pipes#15]
Roadmap

**Comment by ghost** [comment]
don't forget that fluid tanks enlargable with placing them top on top each other :)
or placing them side by side

**Comment by CrafterKina** [comment]
This is implemented on my dev environment. Of course it is extensible.
![2017-01-12_18 34 15](https://cloud.githubusercontent.com/assets/6761325/22060308/8fb1c85e-ddb3-11e6-85a4-2932961069b4.png)

**Comment by ghost** [comment]
are you sure it is extensible? because you placed them top by top, but lava and water stays as they are
i meant; when placing them for exampe side by side two tanks, they inosculates each other automatically, then being a new tank that for example 2x1 block area sized.

**Comment by CrafterKina** [comment]
Here you are.
![test](https://cloud.githubusercontent.com/assets/6761325/22061614/0bafc9d8-ddb9-11e6-9e72-f356ab5fbecb.gif)

**Comment by ghost** [comment]
oh yeah. now i can say that this mod is qualitified :) thanks a lot

**Comment by StevenGann** [comment]
This is very exciting, and looks like it is operational. Any idea when this might be available in a release?

**Comment by ghost** [comment]
it will be probably in the next release.

**Comment by StevenGann** [comment]
Question: Could this work with milk?

**Comment by CrafterKina** [comment]
No, it can't. But if a mod to add milk as a fluid exists, it will work. (Vanilla's milk is not "fluid".)

**Comment by StevenGann** [comment]
Thanks for the answer, now here's another dumb question.

Could a tank of water be used to irrigate farmland?
Would it be possible that pumping bonemeal into a water tank will apply the bonemeal to a nearby farmland block? Each unit of bonemeal could slightly deplete the water tank as well, making farms require a steady supply of water.

This is probably outside the scope of your mod, but I thought I would ask because _I think_ it wouldn't be excessively complicated. As I've probably mentioned before I'm trying to replace the functionality of the CraftBook Bukkit plugin, so forgive me if I pester you with pointless suggestions. I noticed automated crafting on the roadmap, so I thought basic agriculture might be within the same scope.

**Comment by CrafterKina** [comment]
Even assuming I will implement that function, it will not be a function of the tank. I want to keep the tank as simple as a vanilla chest.
